packages/flutter/test/widgets/mouse_region_test.dart

```diff
@@ -1135,6 +1137,97 @@ void main() {
     expect(bottomRegionIsHovered, isFalse);
   });
 
+  testWidgets('Changing MouseRegion\'s callbacks is effective and doesn\'t repaint', (WidgetTester tester) async {
```

Comment:
FYI, you can use double quotes around the string instead of escaping the single quotes here. (not required, but maybe easier to read).